addons/website_blog/static/src/js/website_blog.editor.js

```diff
@@ -72,6 +72,11 @@ if (!$('.website_blog').length) {
 
 const NEW_TAG_PREFIX = 'new-blog-tag-';
 
+// TODO Remove in master.
+for (const el of document.querySelectorAll(".o_wblog_social_links")) {
+    el.classList.add("o_not_editable");
```

Comment:
Why not in the `start()` method?

Reply:
Because `EditorMenu`'s `onWillStart` marks the editable parts as `o_editable` before options get started... which makes the links selectable.